tests/npm-app/build_bundle.js

```diff
@@ -0,0 +1,50 @@
+import { readdirSync, statSync, readFileSync, writeFileSync } from "fs";
+import { join } from "path";
+
+const args = process.argv.slice(2);
+
+const getAllFiles = function (dirPath, arrayOfFiles) {
+  var files = readdirSync(dirPath);
+
+  arrayOfFiles = arrayOfFiles || [];
+
+  files.forEach(function (file) {
+    const filePath = join(dirPath, file);
+    if (statSync(filePath).isDirectory()) {
+      arrayOfFiles = getAllFiles(filePath, arrayOfFiles);
+    } else {
+      arrayOfFiles.push(filePath);
+    }
+  });
+
+  return arrayOfFiles;
+};
+
+const removePrefix = function (s, prefix) {
+  return s.substr(prefix.length);
+};
+
+const rootDir = args[0];
+
+const metadataPath = join(rootDir, "app.json");
+const metadata = JSON.parse(readFileSync(metadataPath, "utf-8"));
+
+const srcDir = join(rootDir, "src", "/");
```

Comment:
The final `/`, also in combination with join, seems really strange.

Reply:
Thanks for the feedback, I've incorporated those suggestions.

This trailing `/` is actually because we want to strip it in `removePrefix`. I've added a comment to clarify that, and removed it from the join.